<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Test</class>
 <widget class="QDialog" name="Test">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>882</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tester ses connaissances</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>181</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>-10</y>
     <width>621</width>
     <height>291</height>
    </rect>
   </property>
   <property name="text">
    <string>Sch√©ma circuit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>520</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>340</y>
     <width>651</width>
     <height>51</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>50</number>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>50</number>
   </attribute>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>20</number>
   </attribute>
   <row>
    <property name="text">
     <string>valeur</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string notr="true">opcode</string>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>accZ</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>acc15</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>selA</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>selB</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>selC</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>pc_ld</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ir_ld</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>acc_ld</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>acc_oe</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>alufs</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>rnw</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="tableWidget_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>400</y>
     <width>651</width>
     <height>51</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>50</number>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>50</number>
   </attribute>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>20</number>
   </attribute>
   <row>
    <property name="text">
     <string>valeur</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string notr="true">opcode</string>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>accZ</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>acc15</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>selA</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>selB</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>selC</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>pc_ld</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ir_ld</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>acc_ld</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>acc_oe</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>alufs</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>rnw</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="tableWidget_3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>280</y>
     <width>651</width>
     <height>51</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>50</number>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>50</number>
   </attribute>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>20</number>
   </attribute>
   <row>
    <property name="text">
     <string>valeur</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string notr="true">opcode</string>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>accZ</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>acc15</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>selA</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>selB</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>selC</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>pc_ld</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ir_ld</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>acc_ld</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>acc_oe</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>alufs</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>rnw</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="tableWidget_4">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>460</y>
     <width>651</width>
     <height>51</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>50</number>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>50</number>
   </attribute>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>20</number>
   </attribute>
   <row>
    <property name="text">
     <string>valeur</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string notr="true">opcode</string>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>accZ</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>acc15</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>selA</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>selB</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>selC</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>pc_ld</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ir_ld</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>acc_ld</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>acc_oe</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>alufs</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>rnw</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>350</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
